app/assets/stylesheets/application.css

```diff
@@ -64,17 +63,59 @@
 }
 
 #flash {
-
+  background: linear-gradient(rgba(198,58,57, .5), rgba(198,58,57, .5));
 }
 
 #space {
 
 }
 
 #pic {
-  height: 100%;
+  padding: 10px;
 }
 
 #title {
-  background-color: #443B3F;
+  background-color: #2e344b;
+}
+
+#gameshow {
+  padding: 50px;
+}
+
+#ha {
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+#form {
+  padding-left: 50px;
+  padding-right: 50px;
+  color: #ffffff;
+}
+
+input[type="button"], input[type="submit"] {
+  border:1px solid #BFBFBF;
+  color: #8C8C8C;
+  border-radius: 30px;
+  -webkit-border-radius: 30px;
+  -moz-border-radius: 30px;
+  font-family: Verdana;
+  width: auto;
+  height: auto;
+  font-size: 16px;
+  padding: 10px 40px;
+  background-color: #FCFAF9;
+}
+
+#pad {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+#igdb {
+
+}
+
+#big {
+  font-size: 2.5em;
 }
```

Comment:
Why so many `id`-specific CSS selectors, instead of reusable classes? You're not putting two elements on a page with the same `id`, right? You would never do that to me.